autovalet_navigation/config/husky/global_costmap_params.yaml

```diff
@@ -2,6 +2,12 @@ global_costmap:
   plugins:
       - {name: static_layer,   type: "costmap_2d::StaticLayer"}
       - {name: inflation,      type: "costmap_2d::InflationLayer"}
+      - {name: lane,           type: "costmap_2d::ObstacleLayer"}
+
+  lane:
+    observation_sources: fake_lane_sensor
+    fake_lane_sensor: {data_type: PointCloud2, sensor_frame: frontCamera_color_optical_frame, clearing: true, marking: true, topic: /lane/pointCloud}
+
   global_frame: map
```

Comment:
will this need to be commented out when doing baseline planner testing? probably not right? just checking...

Reply:
yea I'm not sure whether this should be pushed... this got into my branch while field testing. @SubramanianKrish what's your thoughts?

Reply:
I think we can push it.. Anyways, it's not registering right now as an obstacle on the map. So no harm no foul. Push it and we'll debug using it!